docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,12 @@ develop
     *    - Type
          - Change
 
+    *    - |improved|
+         - A few timelock ops edge cases have been removed. Timelock users must now indicate whether they are booting their
+           servers for the first time or subsequent times, to avoid the situation where a timelock node becomes newly
+           misconfigured and thinks it is booting up for the first time again.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/3498>`__)
+
```

Comment:
nit: The overflow thing is a win and maybe worth calling out (you could say that it's unlikely anyone actually hits this, to be fair, but I think it's still nice)